is_core/templatetags/forms.py

```diff
@@ -12,6 +11,35 @@
 register = template.Library()
 
 
+class FormNode(Node):
```

Comment:
similar implementation as IncludeNode but with exception reraising and little bit simplified